Resources/views/PlatformUI/locationview.html.twig

```diff
@@ -6,8 +6,7 @@
 
 {# using the REST id Location for now to not have to find a cheap and easy way
  # to get the REST id for the regular Location id, but at the some point, this will
- # have to be changed. This involves developping a new view service.
- # see app.renderView() call in WebComponent/ez-subitem.html.twig
+ # have to be changed.
  #}
 <ez-subitem parent-location-id="{{ path( 'ezpublish_rest_loadLocation', {'locationPath': location.pathString|trim('/')} ) }}">
```

Comment:
here you should pass the Location id (not the *REST id*) and the comment above will then be outdated so it can be removed